src/applications/mhv/landing-page/components/HubLinks.jsx

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import recordEvent from '~/platform/monitoring/record-event';
+import { getDestinationDomain } from '../utilities';
 
 const HubSection = ({ title, links }) => {
   const listItems = links.map(({ href, text }, index) => (
     <li key={`${href}--${index}`}>
       <va-link
         class="mhv-c-link"
-        disable-analytics
+        disable-analytics={{
+          value: 'true',
+        }}
```

Comment:
This should be `disable-analyitcs`, one line, no arguments.

![Screenshot from 2023-11-29 12-59-22](https://github.com/department-of-veterans-affairs/vets-website/assets/1299034/8a5a181d-c089-4e7a-9e4e-2a8ea5988287)

Reply:
This fix is in my latest commit. I did see this in storybook but wasn't sure since I couldn't find an example of it in our codebase.